src/main/kotlin/com/msg/gcms/domain/attendance/util/AttendanceConverter.kt

```diff
@@ -0,0 +1,13 @@
+package com.msg.gcms.domain.attendance.util
+
+import com.msg.gcms.domain.attendance.domain.entity.Attendance
+import com.msg.gcms.domain.attendance.domain.entity.Schedule
+import com.msg.gcms.domain.attendance.domain.enums.AttendanceStatus
+import com.msg.gcms.domain.user.domain.entity.User
+
+interface AttendanceConverter {
+    fun toEntity(
+        attendanceStatus: AttendanceStatus = AttendanceStatus.NOT_ATTENDED,
+        user: User, schedule: Schedule
```

Comment:
```suggestion
        user: User,
        schedule: Schedule
```
엇 여기도 줄바꿈이 없는거같네요 의도된거였을까요?